README-dev.md

```diff
@@ -99,46 +99,34 @@ Currently, there are three types of SDK testing: unit, manual, and functional (b
-
-#### Setup
-
-1. There are two environment variables that need to be set in order to publish: `ARTIFACTORY_PYPI_USER` and `ARTIFACTORY_PYPI_PASS`.
-
-	`ARTIFACTORY_PYPI_USER` and `ARTIFACTORY_PYPI_PASS` are one set of credentials used to upload the Python distributions to our internal PyPI repositories. The credentials are the same for both internal PyPI repositories mentioned above.
-
-   - `ARTIFACTORY_PYPI_USER` and `ARTIFACTORY_PYPI_PASS` is the username/password combo given to you by whoever setup your Artifactory pypi account. This is an account separate from your Artifactory account. If you do not have one, please reach out to the `#artifactory` channel and request a `dvp-uploaders-python` account. See <https://docs.delphix.com/pages/viewpage.action?spaceKey=EO&title=Artifactory-instance#Artifactory-instance-SDKpythonpackages> for directions on how to add the account. These are used to upload the Python distributions to our internal PyPI repositories. The credentials are the same for both internal PyPI repositories mentioned above.
-
-2. `twine` needs to be installed. This is a Python package that is used to upload Python distributions. If it's not installed, install it by running `pip install twine`.
+1. Push your code to a branch in the forked repository on Github. Let's say the branch is called `my-feature` in repository called `username/virtualization-sdk`.
+2. Navigate to the app-gate directory and start tests using `git blackbox`. For the guide on which test suite to use,
+see the next sections.
+
+#### Blackbox tests targeting wrappers (mostly Delphix Engine workflows)
+* appdata_python_samples (sample plugins from the app-gate):
+`git blackbox -s appdata_python_samples --extra-params="-p virt-sdk-repo=https://github.com/username/virtualization-sdk.git -p virt-sdk-branch=my-feature"`,
+* appdata_sanity with a direct Python plugin on CentOS 7.3: `git blackbox -s appdata_sanity -c APPDATA_PYTHON_DIRECT_CENTOS73 -a --extra-params="-p virt-sdk-repo=https://github.com/username/virtualization-sdk.git -p virt-sdk-branch=my-feature"`,
+* appdata_sanity with a staged Python plugin on CentOS 7.3: `git blackbox -s appdata_sanity -c APPDATA_PYTHON_STAGED_CENTOS73 -a --extra-params="-p virt-sdk-repo=https://github.com/username/virtualization-sdk.git -p virt-sdk-branch=my-feature"`.
+
+#### Blackbox tests targeting the CLI (~80% CLI tests)
+* virtualization_sdk (installs and tests a direct Python plugin on Ubuntu 18): 
+`git blackbox -s virtualization_sdk -c APPDATA_SDK_UBUNTU18_DIRECT_CENTOS73 --extra-params="-p virt-sdk-repo=https://github.com/username/virtualization-sdk.git -p virt-sdk-branch=my-feature"`,
+* virtualization_sdk (installs and tests a staged Python plugin on Ubuntu 18): 
+`git blackbox -s virtualization_sdk -c APPDATA_SDK_UBUNTU18_STAGED_CENTOS73 --extra-params="-p virt-sdk-repo=https://github.com/username/virtualization-sdk.git -p virt-sdk-branch=my-feature"`.
```

Comment:
same nit as above, use `<username>` to make it clear that it is a placeholder?

Reply:
Done. Zach also approved the blackbox-related readme changes.